File: Garnik/CS50/Week6/Cash/cash.py
from cs50 import get_float, get_int

while True:
    coins = get_float("Change Owed: ")
    if coins > 0:
        break

cents = round(coins * 100)

q = 0

# 25, 10, 5, 1
q = q + cents // 25
cents = cents % 25
q = q + cents // 10
cents = cents % 10
q = q + cents // 5
cents = cents % 5
q = q + cents
print(q)

File: Razmik/Week1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int cash;
    float a;
    do
    {
        a = get_float("Change owed: ");
        cash = round(a * 100);
    }
    while (cash <= 0);
    printf("%i\n", cash / 25 + (cash % 25) / 10 + (cash % 25 % 10) / 5 + cash % 5);
}